deploy/cloudbuild-release.yaml

```diff
@@ -3,4 +3,20 @@ steps:
   - name: "gcr.io/cloud-builders/docker"
     args: ["build", "-f", "deploy/Dockerfile",
            "-t", "gcr.io/kritis-project/kritis-server:$TAG_NAME", "."]
-images: ["gcr.io/kritis-project/kritis-server:$TAG_NAME"]
+  - name: "gcr.io/cloud-builders/docker"
+    args: ["build", "-f", "helm-hooks/Dockerfile",
+           "-t", "gcr.io/kritis-project/preinstall:$TAG_NAME", ".",
+           "--build-arg", "stage=preinstall"]
+  - name: "gcr.io/cloud-builders/docker"
+    args: ["build", "-f", "helm-hooks/Dockerfile",
+           "-t", "gcr.io/kritis-project/postinstall:$TAG_NAME", ".",
+           "--build-arg", "stage=preinstall"]
```

Comment:
We should probably add the predelete image here as well!

Reply:
done